src/entity/SquadPublicRequest.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Column,
+  Entity,
+  Index,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { Source } from './Source';
+import { User } from './user';
+
+enum SquadPublicRequestStatus {
+  Pending = 'pending',
+  Approved = 'approved',
+  Rejected = 'rejected',
+}
+
+@Entity()
+export class SquadPublicRequest {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'text' })
+  @Index('IDX_squad_public_request_sourceId')
+  sourceId: string;
+
+  @ManyToOne(() => Source, {
+    lazy: true,
+    onDelete: 'CASCADE',
+  })
+  source: Promise<Source>;
+
+  @Column({ type: 'text' })
+  requestorId: string;
+
+  @ManyToOne(() => User, {
+    lazy: true,
+    onDelete: 'CASCADE',
+  })
+  requestor: Promise<User>;
+
+  @Column({ default: () => 'now()', type: 'timestamptz' })
+  createdAt: Date;
```

Comment:
On which issue with timestamp and graphorm you are referring to?

Reply:
This rather lengthy thread: https://dailydotdev.slack.com/archives/C06CUKX1P3N/p1715760110448429

Reply:
Not sure of exact cause, but we can fix that with gql field resolvers. Example for user:
```ts
// `src/schema/users.ts`
User: {
  createdAt: (user: Pick<User, 'createdAt'>): Date => {
    return new Date(user.createdAt);
  },
},
```

Reply:
So you can remvoe `type: timestamptz ` from entity.